src/term/mod.rs

```diff
@@ -453,25 +453,24 @@ impl<'a> Iterator for RenderableCellsIter<'a> {
                 (cell, selected)
             };
 
-            let mut invert_selection = false;
             // Set custom selection color and selection text color, if they were provided.
```

Comment:
This comment should be moved to the `let (bg_color, fg_color) = if selected { ...` comment.